sel/sel.cabal

```diff
@@ -33,7 +33,12 @@ common common
     -fprint-potential-instances -Wno-unticked-promoted-constructors
     -Werror=unused-imports
 
-  default-language: Haskell2010
+  default-language:   Haskell2010
+  default-extensions:
+    DerivingStrategies
+    GeneralizedNewtypeDeriving
+    ScopedTypeVariables
+    TypeApplications
```

Comment:
Could we not do this please? Having extensions manually specified per-file is much clearer to a reader.